mapdata/WarcraftLegacies/AbilityData/Skin/961760339.json

```
@@ -0,0 +1,88 @@
+{
+  "OldId": 1718832961,
+  "NewId": 961760339,
+  "Unk": [
+    0
+  ],
+  "Modifications": [
+    {
+      "Id": 1953653089,
+      "Type": 3,
+      "Value": "ReplaceableTextures\\CommandButtons\\BTNEnt.blp"
+    },
+    {
+      "Id": 1718840929,
+      "Type": 3,
+      "Value": "(Druids)"
+    },
+    {
+      "Id": 2037082209,
+      "Type": 3,
+      "Value": "Z"
+    },
+    {
+      "Id": 1835101793,
+      "Type": 3,
+      "Value": "Summon Treants"
+    },
+    {
+      "Id": 2020635233,
+      "Value": 0
+    },
+    {
+      "Id": 1802007137,
+      "Type": 3,
+      "Value": "W"
+    },
+    {
+      "Id": 1953854049,
+      "Type": 3,
+      "Value": "Summons \u003CAOsf,DataB1\u003E Spirit Wolves to fight the Far Seer\u0027s enemies. |nLasts \u003CAOsf,Dur1\u003E seconds. |n|n|cffffcc00Level 1|r - \u003Cosw1,realHP\u003E hit points, \u003Cosw1,mindmg1\u003E - \u003Cosw1,maxdmg1\u003E damage. 2 wolves summoned.|n|cffffcc00Level 2|r - \u003Cosw2,realHP\u003E hit points, \u003Cosw2,mindmg1\u003E - \u003Cosw2,maxdmg1\u003E damage and Critical Strike. 2 wolves summoned.|n|cffffcc00Level 3|r - \u003Cosw3,realHP\u003E hit points, \u003Cosw3,mindmg1\u003E - \u003Cosw3,maxdmg1\u003E damage and Critical Strike. 2 wolves summoned.|n|cffffcc00Level 4|r - \u003Cosw3,realHP\u003E hit points, \u003Cosw3,mindmg1\u003E - \u003Cosw3,maxdmg1\u003E damage, and Critical Strike. 4 wolves summoned.|n|cffffcc00Level 5|r - \u003Cosw3,realHP\u003E hit points, \u003Cosw3,mindmg1\u003E - \u003Cosw3,maxdmg1\u003E damage and Critical Strike. 6 wolves summoned.|n"
+    },
+    {
+      "Id": 1802007905,
+      "Type": 3,
+      "Value": "Z"
+    },
+    {
+      "Level": 1,
+      "Id": 828536161,
+      "Type": 3,
+      "Value": "Summons \u003CSLS9,DataB1\u003E Treants for \u003CSLS9,Dur1\u003E seconds."
+    },
+    {
+      "Id": 2020631137,
+      "Value": 0
+    },
+    {
+      "Level": 2,
+      "Id": 829453409,
+      "Type": 3,
+      "Value": "Summon Treants - [|cffffcc00Level 2|r]"
+    },
+    {
+      "Level": 3,
+      "Id": 829453409,
+      "Type": 3,
+      "Value": "Summon Treants - [|cffffcc00Level 3|r]"
+    },
+    {
+      "Level": 1,
+      "Id": 829453409,
+      "Type": 3,
+      "Value": "Summon Treants - [|cffffcc00Level 1|r]"
+    },
+    {
+      "Level": 2,
+      "Id": 828536161,
+      "Type": 3,
+      "Value": "Summons \u003CSLS9,DataB1\u003E Treants for \u003CSLS9,Dur1\u003E seconds."
+    },
+    {
+      "Level": 3,
+      "Id": 828536161,
+      "Type": 3,
+      "Value": "Summons \u003CSLS9,DataB1\u003E Treants for \u003CSLS9,Dur1\u003E seconds."
+    }
```

It's a bit odd how the cast animation for the trees caused them to momentarily uproot. This fix will change their cast animation to their rooted attack animation:
```suggestion
    },
    {
      "Id": 1768841569,
      "Type": 3,
      "Value": "attack"
    }
```